docker/release/config/opensearch/opensearch-docker-entrypoint.sh

```diff
@@ -66,10 +66,11 @@ if [ -d "$OPENSEARCH_HOME/plugins/$SECURITY_PLUGIN" ]; then
 
     if [ "$DISABLE_SECURITY_PLUGIN" = "true" ]; then
         echo "Disabling OpenSearch Security Plugin"
-        sed "s/plugins.security.disabled.*$/plugins.security.disabled: true" $OPENSEARCH_HOME/config/opensearch.yml | tee $OPENSEARCH_HOME/config/opensearch.yml
+        cat $OPENSEARCH_HOME/config/opensearch.yml | sed "/plugins.security.disabled/d" | tee $OPENSEARCH_HOME/config/opensearch.yml
```

Comment:
Nit; Move the deletion of the line to above 67 so you only have to do it once, then in the if statement append the security setting

Reply:
This is to prevent deleting existing lines if user run the container once, put it down, then restart without changing settings.

Reply:
This script is run on start up and sed runs on both 69 & 73?

Reply:
yes.